e2e/cypress/integration/emoji/custom_emoji_1_spec.js

```diff
@@ -88,7 +88,7 @@ describe('Custom emojis', () => {
         cy.get('.backstage-form__footer').findByText('Cancel').click().wait(TIMEOUTS.FIVE_SEC);
 
         // # Go back to home channel
-        cy.findByText('Back to Mattermost').should('exist').and('be.visible').click().wait(TIMEOUTS.FIVE_SEC);
+        cy.visit('/');
```

Comment:
I am 0/5 on this but do you think we should visit the team channel directly instead of `/${team.name}/channels/off-topic`

Reply:
it's probably a bit more correct yeah, will push up